apps/experiments/views.py

```diff
@@ -39,5 +40,13 @@ def index(request):
     return render(request, "experiments/index.html", dataForRender)
 
 @login_required
-def newE(request):
+def newE(request, id):
+    jsonHeader = request.POST.get("jsonHeader", "")
```

Comment:
Die hier ausgelesenen Daten werden erst auf experiments/new.html eingegeben.
MMn müsste das also nach dem Abschicken der Daten auf experiments/new.html passieren.
Vorschlag: Neue Funktion à la def newESave(request, id) (entsprechend auch Eintrag in urls.py), die dann die Daten ausliest, new_experiment.save() macht und HttpResponseRedirect auf die Experimentdetails des neuen Experiments.

Reply:
Ja kommt noch, nur heute schaffe ich es nicht mehr